harness/propertyHelper.js

```diff
@@ -1,5 +1,6 @@
 
 function isConfigurable(obj, name) {
+    var desc = Object.getOwnPropertyDescriptor(obj, name);
```

Comment:
False. Browser tests for test262 have always run each in a separate iframe. Anything else is not supported.



From: Jordan Harband <notifications@github.com>
Sent: Dec 21, 2015 12:18 PM
To: tc39/test262
Cc: Domenic Denicola
Subject: Re: [test262] Don't stomp on the object in `verifyConfigurable` test. (#452)


In harness/propertyHelper.js<https://github.com/tc39/test262/pull/452#discussion_r48167808>:

> @@ -1,5 +1,6 @@
>
>  function isConfigurable(obj, name) {
> +    var desc = Object.getOwnPropertyDescriptor(obj, name);


Tests can't be run in isolation if they're going to be usefully run in a browser context - this seems like a philosophical concern that affects the entirety of the test262 process.

In other words, it's going to be impractical to run every test in a separate realm in a non-command-line context :-/

-
Reply to this email directly or view it on GitHub<https://github.com/tc39/test262/pull/452/files#r48167808>.